grammars/hrm.cson

```diff
@@ -3,20 +3,20 @@
 'fileTypes': [
   'hrm'
 ]
-'firstLineMatch': '^-- HUMAN RESOURCE MACHINE PROGRAM --$'
+'firstLineMatch': '^ *-- HUMAN RESOURCE MACHINE PROGRAM --'
 'patterns': [
   {
-    'match': '--.*$'
+    'match': ' *--.*'
     'name': 'comment.line.banner.c++'
   }
   {
-    'match': '^    (INBOX|OUTBOX)\\b'
+    'match': '^ *(INBOX|OUTBOX)'
     'captures':
       '1':
         'name': 'entity.name.function.preprocessor.c'
   }
   {
-    'match': '^    (ADD|BUMPDN|BUMPUP|COPYFROM|COPYTO|SUB) +([0-9]+)'
+    'match': '^ *(ADD|BUMPDN|BUMPUP|COPYFROM|COPYTO|SUB) +\\[?([0-9]+)\\]?'
```

Comment:
Okay, last thing, I think…  I'ld like the []s to be coloured like the numbers, since they're part of the argument to the operator.  (I'm happy to fix that when I merge, if you'ld like.)